manifests/case03/configmap/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+bases:
+- ../../base/configmap
```

Comment:
Do we use `yamllint` on YAML files? I see some variation in formatting, may be better to define common configuration for yamllint and use it for all qserv projects? You can probably take one of the existing config as an example and extend it: https://github.com/lsst/daf_butler/blob/main/.yamllint.yaml

Reply:
I'll do it in a future ticket, add it to my TODO list, thanks!